simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -34,71 +34,61 @@
           </mesh>
         </geometry>
       </collision>
```

Comment:
@whispercoros  Can you look at this and make sure it looks good. I have tested it in it works. It is just stereo vision in the sim rather than two cameras